src/ol/color.js

```diff
@@ -102,11 +102,11 @@ export function fromString(s) {
 
   const color = parseRgba(s);
   if (color.length !== 4) {
-    throw new Error('Failed to parse "' + s + '" as color');
+    throw new Error('failed to parse "' + s + '" as color');
```

Comment:
I changed these to lowercase so that they read more naturally when wrapping the messages with more detail (see below).  I could change them back if we want.